cura/Settings/SettingOverrideDecorator.py

```diff
@@ -60,6 +60,17 @@ def __init__(self, *, force_update = True):
         if force_update:
             self._updateNextStack()
 
+    def clear(self) -> None:
```

Comment:
Is this the most common naming for such a method in Python ? I know other languages use e.g. `release` or `dispose`.

Reply:
I changed the name of clear to clearDecoratorData. I believe this gives a better naming for it.